Brain/Brain.ino

```diff
@@ -128,12 +132,9 @@ void loop() {
     board_address = sensor.board_address;
     sensor_id = sensor.id;
     sensorReadFunc(sensor.id);
-    packet = make_packet(sensor.id);
+    packet = make_packet(sensor.id, false);
     Serial.println(packet);
     RFSerial.println(packet);
-    if (!write_to_SD(packet.c_str())){
```

Comment:
Still need to actually writeToSD, just don't do anything w/ the results

Reply:
Oops added that back in